std_msgs/Header header
int32 ID
leg_tracker/LegMsg leg1
leg_tracker/LegMsg leg2
